### ISSUE TITLE
Sign error in the calculation of the errors on the Stokes paramaters 

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#705]
This was pointed out by Fabio in issue #703

I noted something that I can't understand in line 496-497 of kislat2015.py in xStokesAnalysis.normalized_stokes_parameters:

```
QN_ERR[mask] = QN[mask] * numpy.sqrt(Q_REL_ERR**2. + I_REL_ERR**2.)
UN_ERR[mask] = UN[mask] * numpy.sqrt(U_REL_ERR**2. + I_REL_ERR**2.)
```

This means that, when QN or UN are negative, also their error is negative.

**Comment by lucabaldini** [comment]
Pull request: https://github.com/lucabaldini/ixpeobssim/pull/706

**Comment by lucabaldini** [comment]
Shipped in 30.3.1
